addon/object/index.js

```diff
@@ -151,22 +153,24 @@ export const computed = decoratorWithParams(function(target, key, desc, params)
     throw new Error(`ES6 property getters/setters only need to be decorated once, '${key}' was decorated on both the getter and the setter`);
   }
 
-  let value;
-
   if (desc.writable === undefined) {
-    value = {
-      get: desc.get,
-      set: desc.set
-    };
+    const { get, set } = desc;
```

Comment:
Agreed with @kellyselden.  The rule I'd like to follow is roughly: `const` in module scope or for things that are coming from imports, and `let` for everything else. Essentially "liberal let" from [this blog post](https://madhatted.com/2016/1/25/let-it-be).